runtime/common/src/weights/pallet_balances.rs

```diff
@@ -148,4 +148,6 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for WeightInfo<T> {
 		Weight::from_parts(5_873_000, 1501)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 	}
+	fn burn_allow_death() -> frame_support::weights::Weight { todo!() }
+	fn burn_keep_alive() -> frame_support::weights::Weight { todo!() }
```

Comment:
Please run the benchmarks and use proper weights here. If the benches fail for `pallet_balances`, we should fix them in this PR.

Reply:
Good catch, thanks, I'm running them all and will update once they finished.